_data/hosting_providers.json

```diff
@@ -1119,6 +1119,16 @@
     "reviewed": "",
     "note": ""
   },
+  {
+    "name": "Mijn",
+    "link": "https://mijn.host/",
+    "category": "partial",
+    "tutorial": "",
```

Comment:
Is there a tutorial we could put here? We'd like to have tutorials linked to for `partial` entries.